plugins/module_utils/network/sonic/config/mclag/mclag.py

```diff
@@ -193,6 +201,62 @@ def _state_deleted(self, want, have):
                     commands = update_states(diff_want, "deleted")
         return commands, requests
 
+    def _state_replaced(self, want, have, diff):
+        """ The command generator when state is merged
+
+        :rtype: A list
+        :returns: the commands necessary to merge the provided into
+                  the current configuration
```

Comment:
This line can be deleted if the suggested change for the first line is accepted.